Dockerfile-localnet

```diff
@@ -34,11 +50,31 @@ COPY contrib/localnet/zetacored /root/zetacored
 COPY contrib/localnet/tss /root/tss
 
 RUN chmod 755 /root/*.sh
-RUN chmod 700 /root/.ssh
-RUN chmod 600 /root/.ssh/*
 
 WORKDIR /usr/local/bin
-ENV SHELL /bin/sh
 EXPOSE 22
 
-ENTRYPOINT ["/usr/sbin/sshd", "-D"]
+FROM base-runtime AS latest-runtime
+
+COPY --from=cosmovisor-build /go/bin/cosmovisor /usr/local/bin
+COPY --from=latest-build /go/bin/zetacored /go/bin/zetaclientd /go/bin/zetaclientd-supervisor /go/bin/zetae2e /usr/local/bin
+
+# optional old version build
```

Comment:
Since we're removing `upgrade` Dockerfile, it might be good to add some comments adding context for having the old version build here: would be used to test upgrade locally

Reply:
done